api/app/Repositories/EventType/EventTypeRepositoryInterface.php

```diff
@@ -13,4 +13,6 @@ public function getById(int $id): ?EventType;
     public function save(EventType $eventType): EventType;
     public function deleteById(int $id): void;
     public function getEventTypesByOwnerId(int $id): Collection;
+    public function saveAvailabilities(EventType $eventType, array $availabilities): void;
+    public function deleteAvailabilities(EventType $eventType): void;
```

Comment:
Probably it is better to put the methods to the separate `AvailabilityRepository`, but ok for now.